neurom/fst/_mm.py

```diff
@@ -299,6 +294,17 @@ def section_area(section):
     return map_sum_segments(mm.segment_area, section)
 
 
+def section_tortuosity(section):
+    '''Tortuosity of a section
+
+    The tortuosity is defined as the ratio of the path length of a section
+    and the euclidian distnce between its end points.
+
```

Comment:
So, maybe we should update the doc-string: 

The tortuosity is defined as the ratio between the actual length of a  section and the distance between its end points.